src/components/InputTypes/index.tsx

```diff
@@ -0,0 +1,71 @@
+import Checkbox from "./Checkbox";
+import Code from "./Code";
+import Radio from "./Radio";
+import Range from "./Range";
+import Select from "./Select";
+import ShortText from "./ShortText";
+import TextArea from "./TextArea";
+
+import React, { ChangeEvent } from "react";
+
+import { QuestionType } from "../../api/question";
+import { QuestionProp } from "../Question";
+
+const _require_options: Array<QuestionType> = [
```

Comment:
Not sure the leading underscore is necessary here. It's only really relevant in languages where everything is exported.

```suggestion
const require_options: Array<QuestionType> = [
```